modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/integration/OpenApiServlet.java

```diff
@@ -80,4 +107,43 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         }
 
     }
+
+    public static MultivaluedHashMap<String, String> getQueryParams(String queryString) {
+        MultivaluedHashMap<String, String> queryParameters = new MultivaluedHashMap<>();
+
+        if (StringUtils.isEmpty(queryString)) {
+            return queryParameters;
+        }
+
+        String[] parameters = queryString.split("&");
+
+        for (String parameter : parameters) {
+            String[] keyValuePair = parameter.split("=");
```

Comment:
Query parameters could only consist of a key (`https://example.com/path?my-parameter`) but we're only handling key/value pairs here.